src/Movement/Kinematics/HangprinterKinematics.cpp

```diff
@@ -342,32 +402,33 @@ inline float HangprinterKinematics::MotorPosToLinePos(const int32_t motorPos, si
 
 void HangprinterKinematics::flexDistances(float const machinePos[3], float const distanceA,
```

Comment:
This function must take an array of numAnchors distances that must be looped over

Reply:
Yes, adapt everything in #569 to not using A_AXIS constants inside.
But first deal with HangprinterKinematics::StaticForces and the other changes in #562